src/box/lua/config/applier/box_cfg.lua

```diff
@@ -220,10 +220,10 @@ local function names_schema_upgrade_on_replace(old, new)
         return
     end
 
-    local latest_version = mkversion.get_latest()
+    local version = mkversion(2, 11, 5)
```

Comment:
Nit: `required_version`, `wanted_version`, `needed_version`, `expected_version`, `waited_version` -- I would use something like this to make it clear that it is not some current schema verison.